templates/full-time-program.html

```diff
@@ -52,8 +52,9 @@ <h1 class="blue-h1">The Program</h1>
       </div>
       <p class="justify">
         The program usually runs Monday through Friday, 8:00 AM to 5:00 PM (Pacific Time)
-        for six months (from January to June and July to December). The whole program is
-        a year long, starting with training at Techtonica, and followed by six months of
+        for six months (from January to June and July to December). The July 2024 cohort will
+        be a five month cohor, running from July to November. The whole program is
+        about a year long, starting with training at Techtonica, and followed by six months of
         remote job placement or job search and community support. After the first trial month of
         full-time training, participants who work the hardest and meet and exceed
```

Comment:
After "support" is where we should put the three tuition-related paragraphs from G2, G3, and G4. And let's just remove the "After the first trial month of full-time learning" and start with "Participants who work hard and meet and exceed expectations will be allowed to continue the program" without "for the next five months of training with stipends based upon need and funding available" and changing "We still recommend you save up as much as possible as stipends may not cover all your costs, and we" to "We do not recommend working during the program. We do not provide medical . . ."

Reply:
### ✅ Addressed in [f5a0daf](https://github.com/Techtonica/techtonica.org/pull/336/commits/f5a0daf59b8f915df48c3b6f5763dc25ed05dbd7).